pinot-core/src/test/java/org/apache/pinot/queries/FastHllQueriesTest.java

```diff
@@ -240,7 +240,7 @@ private void buildAndLoadSegment(boolean hasPreGeneratedHllColumns)
     // range. For this test, we first need to fix the input avro data
     // to have the time column values in allowed range. Until then, the check
     // is explicitly disabled
-    segmentGeneratorConfig.setCheckTimeColumnValidityDuringGeneration(false);
+    segmentGeneratorConfig.setSkipTimeValueCheck(true);
```

Comment:
If we really go with default as false (check disabled), we really don't need to do setSkipTimeValueCheck(true) in any of the tests to skip the check.

Reply:
We still enable the check by default, it is just the boolean value default is false.